src/sentry/services/hybrid_cloud/integration/service.py

```diff
@@ -132,19 +128,6 @@ def get_organization_integration(
         )
         return ois[0] if len(ois) > 0 else None
 
-    def find_pagerduty_service(
-        self, *, organization_id: int, integration_id: int, service_id: Union[str, int]
-    ) -> Optional[PagerDutyServiceDict]:
-        org_integration = self.get_organization_integration(
-            integration_id=integration_id, organization_id=organization_id
-        )
-        if not org_integration:
-            return None
-        try:
-            return OrganizationIntegration.find_service(org_integration.config, service_id)
-        except StopIteration:
-            return None
-
```

Comment:
We're not rolled out so it doesn't matter, but in the future would we be able to whole cloth remove RPC services like this? This gets me thinking about deprecation strategies for stuff like this down the line 🤔

Reply:
> We're not rolled out so it doesn't matter, but in the future would we be able to whole cloth remove RPC services like this? 

No we won't be able to remove RPC methods soon. Thankfully this wasn't an RPC method.

In the future we'd need to stop calling an RPC method, deploy that change, then deploy a separate change to remove the RPC method. Much like migrations removals will always need to be done over multiple deploys.